src/libraries/System.Net.Http/tests/FunctionalTests/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "system.net.websockets.client.tests",
+  "private": true,
+  "dependencies": {
+    "node-abort-controller": "^3.0.1",
```

Comment:
We need exact versions. Also please check their dependencies, if they are safe for CI.

Reply:
There are 3 dependencies 
- [whatwg-url](https://www.npmjs.com/package/whatwg-url)
- [webidl-conversions](https://www.npmjs.com/package/webidl-conversions)
- [tr46](https://www.npmjs.com/package/tr46)

Audit found 0 vulnerabilities.